test/test_core.py

```diff
@@ -4177,6 +4179,21 @@ def test_dylink_safe_heap(self):
     self.set_setting('SAFE_HEAP')
     self.do_basic_dylink_test()
 
+  @needs_dylink
+  def test_dylink_locate_file(self):
+    name = 'liblib.so'
```

Comment:
Maybe call this `so_name` for consistency?

Reply:
Done. I've rebased with the changes.